src/app/components/add-card/add-card.component.css

```diff
@@ -0,0 +1,13 @@
+.date {
+  position: relative;
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #aaacbc;
+}
+
+ul li {
+  list-style: none;
+}
```

Comment:
Make this style global. Perhaps add Bulma or bootstrap to the project?

Reply:
made it global.